_data/issues/validation.yml

```diff
@@ -0,0 +1,13 @@
+title: Validation
+alert_type: info
+items:
+- date: 2023-11-28
+  summary: Screen reader and screen magnification users did not notice when the validation status changed.
+  summary_additional:
+  issue_repo: uswds
+  issue_number: 5642
+- date: 2023-10-25
+  summary: Users are confused about the purpose of the validation component.
+  summary_additional: The component does not match user expectations for how their information should be validated in a form input.
+  issue_repo: uswds-site
+  issue_number: 2320
```

Comment:
Yes the issue is about describing the problems users encountered with validation which lead to our needing to re-evaluate the use case of the component, not communicating the proper use case. I think maybe this was just put in the wrong repo maybe? Do you think I should move to uswds repo?

Reply:
I vote for moving it to the uswds repo. Wondering if it would be helpful to re-title the issue as well to something like "USWDS-Site - Validation: Evaluate use case"

Reply:
Done - thank you!

Reply:
Thanks! I updated the entry in [ed22720](https://github.com/uswds/uswds-site/pull/2402/commits/ed2272037db709cf9a32b673e4b3a3efcce8773e)